internal/broadcast/message.go

```diff
@@ -19,31 +19,40 @@ package broadcast
 import (
 	"context"
 
-	"github.com/hyperledger-labs/firefly/internal/config"
 	"github.com/hyperledger-labs/firefly/internal/i18n"
 	"github.com/hyperledger-labs/firefly/internal/log"
 	"github.com/hyperledger-labs/firefly/pkg/database"
 	"github.com/hyperledger-labs/firefly/pkg/fftypes"
 )
 
 func (bm *broadcastManager) BroadcastMessage(ctx context.Context, ns string, in *fftypes.MessageInOut, waitConfirm bool) (out *fftypes.Message, err error) {
-	return bm.BroadcastMessageWithID(ctx, ns, nil, in, nil, waitConfirm)
+	return bm.BroadcastMessageWithID(ctx, ns, nil, in, nil, waitConfirm, false)
 }
 
-func (bm *broadcastManager) BroadcastMessageWithID(ctx context.Context, ns string, id *fftypes.UUID, unresolved *fftypes.MessageInOut, resolved *fftypes.Message, waitConfirm bool) (out *fftypes.Message, err error) {
+func (bm *broadcastManager) BroadcastMessageWithResolvedID(ctx context.Context, ns string, in *fftypes.MessageInOut, waitConfirm bool) (out *fftypes.Message, err error) {
+	return bm.BroadcastMessageWithID(ctx, ns, nil, in, nil, waitConfirm, true)
+}
+
+// THIS IS WHERE THE NEW CODE FIRST CALLS ResolveInputIdentity
+// IT already has an author and key here
```

Comment:
This comment feels like a note to the developer and maybe not something that should be committed... or at least it should be cleaned up before being committed.